src/context.ts

```diff
@@ -37,16 +38,19 @@ export class Context<RequestParamKeyType extends string = string, E = Env> {
     }
     this.event = event
     this.notFoundHandler = notFoundHandler
+    this.finalized = false
+  }
 
-    if (!this.res) {
-      const res = new HonoResponse(null, { status: 404 })
-      res._finalized = false
-      this.res = res
-    }
+  get res(): Response {
```

Comment:
By this implementation, `HonoResponse` is not necessary. Right?

Reply:
I thought so.

Reply:
That's good!